querybook/webapp/components/DataTableStats/DataTableColumnStats.tsx

```diff
@@ -1,27 +1,19 @@
 import * as React from 'react';
 
-import { useResource } from 'hooks/useResource';
+import { ITableColumnStats } from 'const/metastore';
 import { isNumeric } from 'lib/utils/number';
-import { TableColumnResource } from 'resource/table';
 import { KeyContentDisplay } from 'ui/KeyContentDisplay/KeyContentDisplay';
 
 import { TableStats } from './DataTableStatsCommon';
 
 interface IProps {
-    columnId: number;
+    stats: ITableColumnStats[];
 }
 
 export const DataTableColumnStats: React.FunctionComponent<IProps> = ({
-    columnId,
+    stats,
 }) => {
-    const { data: tableColumnStats } = useResource(
```

Comment:
since we don't get tags, data elements, and stats individually anymore, can you remove the api and resource?

Reply:
actually there are 3 `/column/stats` apis, two of them are also not used today. should we remove all of them?

Reply:
oh yeah lets remove all of them then